app/src/main/java/chat/rocket/android/members/presentation/MembersPresenter.kt

```diff
@@ -21,30 +23,29 @@ class MembersPresenter @Inject constructor(
     private val view: MembersView,
     private val navigator: ChatRoomNavigator,
     private val dbManager: DatabaseManager,
+    private val permissionsInteractor: PermissionsInteractor,
     @Named("currentServer") private val currentServer: String,
     private val strategy: CancelStrategy,
     private val mapper: MemberUiModelMapper,
     val factory: RocketChatClientFactory,
     private val userHelper: UserHelper
 ) {
     private val client: RocketChatClient = factory.get(currentServer)
-    private var offset: Long = 0
 
     /**
      * Loads all the chat room members for the given room id.
      *
      * @param roomId The id of the room to get chat room members from.
      */
-    fun loadChatRoomsMembers(roomId: String) {
+    fun loadChatRoomsMembers(roomId: String, offset: Long = 0, clearDataset: Boolean = false) {
```

Comment:
What is the reason of adding the offset param on the function declaration here?

Reply:
Actually there is one bug in MembersFragment where members are not loaded as recycler view is scrolled using EndlessRecyclerViewListener. And only one page of member is shown.

To fix that I followed same pattern we are using for EndlessRecyclerView in ChatRoomFragment
https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt#L810

Reply:
It is fixed now - you can test by seeing a channel with a lot of users.